src/substrait/json.py

```diff
@@ -0,0 +1,25 @@
+from google.protobuf import json_format
```

Comment:
There are three Google ways of loading/saving protobuffers:

binary
JSON format
text format

It's worth considering how you want to handle the four different types (including the Substrait text format).  The C++ package implements methods to read/write any.